src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -42,11 +42,19 @@ object IndexConstants {
   val INDEX_HYBRID_SCAN_DELETE_ENABLED = "spark.hyperspace.index.hybridscan.delete.enabled"
```

Comment:
Do we need to keep around this config?

Reply:
If we introduce other threshold configs for hybrid scan, this config will make things simple. I think we could keep this config until the feature is stable.

Reply:
OK. If you are thinking through this, then we can leave it for now.